agent/php_nrini.c

```diff
@@ -3100,6 +3100,18 @@ STD_PHP_INI_ENTRY_EX("newrelic.vulnerability_management.composer_api.enabled",
                      newrelic_globals,
                      nr_enabled_disabled_dh)
 
+/*
+ * Cloud relationship settings
+ */
+STD_PHP_INI_ENTRY_EX("newrelic.cloud.aws.account_id",
+                     "1",
+                     NR_PHP_REQUEST,
+                     nr_string_mh,
```

Comment:
Can this call a function that verifies the string is always 12 chars long (according to the spec).

Reply:
added in [5c45a0e](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f)

Reply:
Can you please add a test case to verify the functionality?

Reply:
tests were added in [25475ef](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/25475efa60752f455c8ca6bf90d5ef8e15ed1cc0)

Reply:
Nice!